pkg/fsp/header_test.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2017-2018 the LinuxBoot Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package fsp
+
+import (
+	"bytes"
+	"io/ioutil"
+	"testing"
+)
+
+const (
+	// From https://github.com/IntelFsp/FSP/blob/master/ApolloLakeFspBinPkg/FspBin/Fsp.fd
+	// under the FSP license. See README.md under `test_blobs`.
+	FSPTestFile = "test_blobs/ApolloLakeFspBinPkg/Fsp.fd"
+)
+
```

Comment:
Would be nice to have some tests for the attribute parsing. I think this could be improved by moving the attribute parsing to separate functions and testing those. Another test that would be nice is just to try to read a badly formatted fsp binary and check for the error thrown

Reply:
ah now I understand what you meant above about splitting attribute parsing.

Do you have something like this in mind?
```
func (ca ComponentAttribute) Parse() []string {
    // parser and return an array of strings, e.g. []string{"ReleaseBuild", "OfficialRelease"}
}
```